app/helpers/display_ad_helper.rb

```diff
@@ -0,0 +1,6 @@
+module DisplayAdHelper
+  def display_ads_placement_area_options_array
+    DisplayAd::ALLOWED_PLACEMENT_AREAS.map
+      .with_index { |area, index| [DisplayAd::ALLOWED_PLACEMENT_AREAS_HUMAN_READABLE[index], area] }
```

Comment:
Ruby has this built-in:

```suggestion
    DisplayAd::ALLOWED_PLACEMENT_AREAS_HUMAN_READABLE.zip(DisplayAd::ALLOWED_PLACEMENT_AREAS)      
```

You might also consider making `ALLOWED_PLACEMENT_AREAS_HUMAN_READABLE` part of the helper.